jormungandr/src/blockchain/candidate.rs

```diff
@@ -49,7 +48,7 @@ mod chain_landing {
 
     impl<S> State<S>
     where
-        S: Stream<Item = Result<Header, Error>> + Unpin,
+        S: Stream<Item = Header> + Unpin,
```

Comment:
Hmm, I would expect the network streams to be fallible (items like `Result<Header, Error>`) here and below?

Reply:
I'm wrong, these are internal streams.